src/System.IdentityModel.Tokens/Utility.cs

```diff
@@ -65,5 +65,25 @@ internal static string SerializeAsSingleCommaDelimitedString(IEnumerable<string>
 
             return sb.ToString();
         }
+
+        public static bool IsHttps(string address)
+        {
+            if (string.IsNullOrEmpty(address))
+            {
+                return false;
+            }
+
+            return address.StartsWith("https");
+        }
+
+        public static bool IsHttps(Uri uri)
+        {
+            if (uri == null)
+            {
+                return false;
+            }
+
+            return uri.Scheme == "https";
```

Comment:
UriSchemeName is made internal in dnxcore: https://github.com/dotnet/corefx/blob/41e203011152581a6c65bb81ac44ec037140c1bb/src/System.Private.Uri/src/System/Uri.cs